test/rpc_test.exs

```diff
@@ -165,7 +165,7 @@ defmodule RPCTest do
       # %Tds.Parameter{direction: :input, name: "@8", type: :uuid, value: nil}]
     ] 
     assert [{1, nil}] = query(sql, params)
-    assert :ok = query("DROP TABLE dbo.TestTable", [])
+    assert :ok = query("DROP TABLE posts", [])
```

Comment:
line 129 created table "posts" at the beginning of the test we're closing here, "TestTable" is used elsewhere.